standards_lab/api/management/commands/migratedata.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from utils.project import get_project_config
+import os
+
+
+class Command(BaseCommand):
+    help = "Migrates files in storage to a new layout"
+
+    def add_arguments(self, parser):
+        pass
+
+    def handle(self, *args, **options):
+        project_names = os.listdir(settings.ROOT_PROJECTS_DIR)
+        for project_name in project_names:
+            print("Project: " + project_name)
+            project = get_project_config(project_name)
+            datas_dir = os.path.join(settings.ROOT_PROJECTS_DIR, project_name, "datas")
```

Comment:
I've gone for datasets, cos I think it's good to avoid the existing uploadType values - we may get in a situation where sometimes the dir names are the same as uploadType values and sometimes they are different and that's extra mental load for a coder and a possible source of bugs later. If we just keep them different all the time it's clearer.